pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.black]
+
+include = '''
+.*\.py$
+'''
+
+force-exclude = '''
+^(?!\/teams\/).*
+'''
+
+
```

Comment:
so that users using [black](https://black.readthedocs.io/en/stable/index.html) for formatting won't start reformatting the unformatted main files.